plugins/synaptics-rmi/fu-synaptics-rmi-firmware.c

```diff
@@ -510,6 +511,12 @@ fu_synaptics_rmi_firmware_get_sig_size (FuSynapticsRmiFirmware *self)
 	return self->sig_size;
 }
 
+guint32
+fu_synaptics_rmi_firmware_get_firmware_size (FuSynapticsRmiFirmware *self)
+{
+	return self->firmware_size;
+}
+
```

Comment:
I wonder if it's going to be better to rebase this on https://github.com/fwupd/fwupd/pull/2998 when that goes in.